app/src/main/res/layout/fragment_spaces_parking_dialog.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/teal_200"
+    tools:context=".fragment.SpacesParkingDialogFragment">
+
+    <TextView
+        android:id="@+id/text_viewdialog_fragment_amount_spaces"
+        style="@style/TextViewFragmentStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:text="@string/text_view_dialog_fragment_spaces"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
```

Comment:
in this EditText you can add the  gravity and the inputType to the style

Reply:
done